roles/pulp/templates/pulp.container.j2

```diff
@@ -0,0 +1,17 @@
+# [Unit]
```

Comment:
Using this method, is this file needed?

Reply:
This one is definitely not needed here. I intend to drop this.